stable/prometheus-redis-exporter/values.yaml

```diff
@@ -54,6 +54,48 @@ serviceMonitor:
   # Set of labels to transfer on the Kubernetes Service onto the target.
   # targetLabels: []
 
+## Custom PrometheusRules to be defined
+## The value is evaluated as a template, so, for example, the value can depend on .Release or .Chart
+## ref: https://github.com/coreos/prometheus-operator#customresourcedefinitions
+prometheusRule:
+  enabled: false
+  # additionalLabels: {}
```

Comment:
Should we uncomment these 3 lines? 
Just trying to match what I see in https://github.com/helm/charts/blob/beb56b82dfc9a02fc51edd45b8759405194708bc/stable/redis/values.yaml#L524-L528

Reply:
We can of course just uncomment them. In other charts these parameters are uncommented, too. The chart works fine without issues either way as far as I can tell, but I think empty default values are better than undefined keys.